src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public enum Colors {
+        RED,
+        BLUE,
+        YELLOW,
+        WHITE,
+        BLACK,
+        PURPLE,
+        ORANGE
```

Comment:
The enum name 'Colors' should be in singular form according to Java naming conventions, as each instance of an enum is a single color. Consider renaming it to 'Color'.